serdes/protobuf-serde/pom.xml

```diff
@@ -25,6 +25,11 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.squareup.wire</groupId>
+            <artifactId>wire-schema</artifactId>
+        </dependency>
```

Comment:
What does this do?  This is one of those dependencies that dependabot keeps wanting to update but I don't know what it's for so I haven't merged the dependabot PR yet.  Can you TLDR it for me?

Reply:
look at `FileDescriptorUtils` class.
That dependency provides `ProtoFileElement` class, this class is the only way I found to get a textual representation of a protobuf schema

Reply:
OK so the wire-schema dependency provides all that file descriptor stuff.  Any idea whether https://github.com/Apicurio/apicurio-registry/pull/1201 is OK to merge (once the artifact-groups branch is merged)?